agentops/partners/langchain_callback_handler.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         endpoint: Optional[str] = None,
         max_wait_time: Optional[int] = None,
         max_queue_size: Optional[int] = None,
-        tags: Optional[List[str]] = None,
+        default_tags: Optional[List[str]] = None,
     ):
 
         logging_level = os.getenv("AGENTOPS_LOGGING_LEVEL")
```

Comment:
Probably don't need this anymore as a bunch of __init__.py functions does this already